Description: Make Xsession.d script ignore errors.
Author: Michael Meskes <meskes@debian.org>

Index: virtualbox/src/VBox/Additions/x11/Installer/98vboxadd-xclient
===================================================================
--- virtualbox.orig/src/VBox/Additions/x11/Installer/98vboxadd-xclient
+++ virtualbox/src/VBox/Additions/x11/Installer/98vboxadd-xclient
@@ -44,13 +44,13 @@ elif test -z "${SSH_CONNECTION}"; then
     # Check wheather X11 or Wayland version of VBoxClient should be started.
     vbox_wl_check=$(/usr/bin/vboxwl --check 2> /dev/null)
     if test "$vbox_wl_check" = "WL"; then
-        /usr/bin/VBoxClient --wayland
+        /usr/bin/VBoxClient --wayland || true
     else
-        /usr/bin/VBoxClient --clipboard
-        /usr/bin/VBoxClient --seamless
-        /usr/bin/VBoxClient --draganddrop
+        /usr/bin/VBoxClient --clipboard || true
+        /usr/bin/VBoxClient --seamless || true
+        /usr/bin/VBoxClient --draganddrop || true
     fi
 
-    /usr/bin/VBoxClient --checkhostversion
-    /usr/bin/VBoxClient --vmsvga-session # In case VMSVGA emulation is enabled
+    /usr/bin/VBoxClient --checkhostversion || true
+    /usr/bin/VBoxClient --vmsvga-session || true # In case VMSVGA emulation is enabled
 fi
